images/linux/scripts/base/apt-mock.sh

```diff
@@ -14,15 +16,35 @@ while [ \$i -le 30 ];do
   err=\$(mktemp)
   $real_tool "\$@" 2>\$err
   result=\$?
-  if [ \$result -eq  0 ];then
-    break
+  cat \$err >&2
+
+  # no errors, continue
+  test \$result -eq  0 && continue
```

Comment:
Why it is not `break`?
Looks like with `continue` script will retry command even if it is finished with exit code 0

Reply:
Also typo double space: `\$result -eq  0`